app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (
+    NotWearingMaskError,
+    NotVaccinatedError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
```

Comment:
You have used type hinting for `visitor` as a `dict` which is great. But it would be even better if you could specify what type of values this dictionary should contain. For example, you might use `Dict[str, Union[str, Dict[str, datetime.date]]]`. This would make your code more self-documenting and help other developers understand what kind of data they should pass to this function.